README.md

```diff
@@ -814,6 +814,21 @@ const MyConnectedComponent = connect(
 )(MyComponent);
 ```
 
+### Hot reloading
+
+Given a Found-enhanced Redux store, you can dynamically replace the route configuration with `found.replaceRouteConfig` on the store object. This will replace the route configuration and run the match with the current location against the new route configuration.
+
+```js
+if (module.hot) {
+  module.hot.accept('./routeConfig', () => {
```

Comment:
how would this look with rhl4? can the `hot` helper be used?

Reply:
no, that's just for react components. this needs to update something in the store. it's like `replaceReducer`.